app/build.gradle

```diff
@@ -1,6 +1,9 @@
 apply plugin: 'com.android.application'
 
 android {
+    testOptions {
+        unitTests.returnDefaultValues = true
```

Comment:
Can you enlighten me on what this configuration does?

Reply:
I was trying to get my tests to run but it did not, I have taken it out thanks for pointing that out.